packages/reporter/src/runnables/runnables.tsx

```diff
@@ -1,21 +1,17 @@
 import _ from 'lodash'
 import { action } from 'mobx'
 import { observer } from 'mobx-react'
-import React, { Component } from 'react'
+import React, { Component, MouseEvent } from 'react'
+import Divider from '@material-ui/core/Divider'
```

Comment:
Just a heads up - this isn't actually a library that we use in the TR. This works locally since we use `@material-ui` for a demo within the independent `@cypress/react` package and it gets installed in the root directory but it's not going to work in prod